src/main/java/reposense/RepoSense.java

```diff
@@ -110,6 +113,8 @@ public static void main(String[] args) {
             FileUtil.zipFoldersAndFiles(reportFoldersAndFiles, cliArguments.getOutputFilePath().toAbsolutePath(),
                     ".json");
 
+            GitConfig.setGlobalGitLfsConfig(globalGitConfig);
```

Comment:
Not sure why this is here since the git cloning would have been done before this line.

Reply:
I take this back.